### GeneNetwork News

 **2019-Sep-24: BXD Genotypes file: Sequence-based improvements coming
 soon**

 The BXD sequencing consortium has generated full genome sequence for
 152 extant BXD strains. Dr. David Ashbrook has extracted the most
 informative 40,000 SNPs from sequence data, and compared
 sequence-based genotypes with those based on array platforms (see
 entries below).  In general there is excellent concordance for all but
 a handful of the BXD strains that are in widespread use (BXD1 to
 BXD102). However, there are some problems and mixups that we list
 below. We are currently working on harmonizing data when possible, and
 understanding and fixing errors, when not.

 This evaluation of genotypes is based on data extracted from whole-genome sequencing of BXD male spleen samples obtained in January 2017 from the Jackson Laboratory whenever possible, and from the UTHSC colony in all other cases. All animals were born between January and December 2016.

1. BXD65b (sourced from JAX). The sequence-based genotypes correlate better to array-based 2017 genotypes of BXD65 and BXD65a than with the BXD65b array-based genotypes. It is possible that there are two somewhat distinct sub-strains now called BXD65b—one at JAX and one at UTHSC. **Recommended Action:**Sourcing BXD65b animals from JAX.

2. BXD53 array-based genotypes do not correlate well with sequence-based genotypes for BXD53/2. BXD53 was rederived a second time by JAX and is now referred to formally at BXD53/2RwwJ (Stock 017749). The likely reason for the discrepancy is that we sequenced BXD53/2RwwJ (Stock 017749) rather than the older BXD53 (sourced from UTHSC) for array-based genotyping. BXD53 sequence-based genotypes correlate well with eQTL data for BXD53. **Recommended Action:**If using BXD53/2RwwJ, please obtain current sequence-based genotypes from David Ashbrook. 

3. BXD68 animals at UTHSC used from about 2012 to 2017 were BXD34 animals. Genotypes in the 2017 array-based file HAVE or HAVE  NOT been corrected. We have corrected this error in some phenotype data sets. BXD68 from The Jackson Laboratory are genuine BXD68. We sequenced BXD68 DNA from JAX. **Recommended Action: **None unless you have used BXD68 animals obtained from UTHSC between 2012 and 2017. If so, these animals should be relabeled BXD34.

4. BXD81 sequence-based genotypes correlate well with the new array-based genotypes (r = 0.81) but less well with old array-based genotypes (r = 0.66). This is presumably due to more complete inbreeding. The new sequence-based genotypes are for BXD81 sources from JAX. **Recommended Action: **None 

5. BXD88 sequence-based genotypes do not correlate well with either of the two array-based genotype files (r = 0.50 and 0.54). Like BXD53, BXD88 was rederived a second time by JAX and is now referred to formally at BXD88/2RwwJ (Stock 007110). BXD88 was admixed at some point in the last decade that has since been rederived. BXD88/2RwwJ is only available at JAX. **Recommended Action:**Sourcing BXD88/2RwwJ from JAX; do not treat as equivalent to old BXD88. 

6. BXD100 (JAX stock 007143) sequence-based genotypes correlates well with the older array-based genotypes (r = 0.96) but not so well with newer 2017 array data (r = 0.55). This suggests some admixture in the last decade at UTHSC that has since been fixed. **Recommended Action:**If you used BXD100s between 2006 and 2014, we recommend using genotypes from the older array-based file for this strain. If you are using new BXD100 animals sourced from JAX, then use the newer 2017 array-based genotypes or the sequence-based genotypes.

7. BXD151 (UTHSC stock) sequence-based genotypes do not correlate with the array-based genotypes at all (r = -0.79 and -0.06). BXD151 sequence-based genotypes correlate very well with older array-based sequence for the now (formally) extinct BXD128a (r = 0.97 and 0.99 with the two array-based files). We need to check sequence of BXD151/RwwJ (JAX 029868). **Recommended Action:**None. It appears likely that BXD151 may be equivalent to what we used to call BXD128a. BXD128a is no longer officially available and we will retain the name BXD151.

8. BXD155 (UTHSC stock) sequence-based genotypes do not correlate with arrays (r = 0.21 and 0.18). BXD155 has an excellent correlation with previous array-based genotypes for strain BXD152 (r = 0.95 and 0.95 with arrays, 0.99 with sequence). Likely reading error or strain swap at UTHSC. BXD152 (Stock 029869, non-agouti black coat color) can be obtained from JAX, but BXD155 (gray coat color) is not yet available from JAX. **Recommended Action:**Check with David Ashbrook or Rob Williams on whether this issue has been resolved. It seems likely that the BXD155 sample that we sequenced was actually at BXD152.

9. BXD162 (UTHSC stock) sequence-based genotypes correlate poorly well with new array genotypes (r = 0.65 and 0.69). This strain also has a reasonable correlation with BXD170 in older array genotypes (r = 0.7). Possible admixture. BXD162 is not yet available from JAX. **Recommended Action:**Check with David Ashbrook or Rob Williams in you use this strain regarding correct nomenclature.

10. BXD194 (UTHSC stock) sequence-based genotypes correlated with BXD87 (JAX stock) sequence-based genotypes. Probably a sample error at UTHSC when preparing samples for sequencing. We need to check the JAX BXD194/RwwJ (Stock 030878). **Recommended Action:**Check with David Ashbrook or Rob Williams in you use this strain regarding correct nomenclature.

 **2018-12-19: BXD Genotypes file status (December 2018)**

 In September and October 2016, Robert Williams, Jesse Ingels, Lu Lu,
 and Danny Arends released new genotype files for many of the original
 BXD strains (BXD1 through BXD102), and for all of the new strains
 (BXD104 to BXD220). Genotypes were generated at about 74,000 SNPs
 using the GigaMUGA array developed by Drs. Fernando Pardo-Manuel de
 Villena (University of North Carolina) and Gary Churchill (The Jackson
 Laboratory). Genotypes were generated at GeneSeek (Neogen Inc) with
 financial support from the University of Tennessee Center for
 Integrative and Translational Genomics.

 **Update**The new genotypes are now available in GeneNetwork as the
 [2019 Genotype
 file](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=600).
 All SNPs were mapped to the older July 2007 mm9 NCBI Build 37 assembly
 and to the newer Dec 2011, mm10, GRCm38 assembly. Of the 74,000
 GigaMUGA and many other markers we have typed, only about 6800 are
 useful (informative) in defining recombination events in the current
 set of BXDs. These informative markers either define unique
 recombination patterns across the 198 BXD strains (including extinct
 strains) or they define the proximal and distal ends of regions that
 do not contain any known recombinations in the BXD family.

 This file provides consensus genotypes for 198 BXD strains and for the
 two progenitor strains and the reciprocal F1s. Of the 198 BXD strains,
 191 are independent, whereas 7 are substrains (e.g., BXD48 and
 BXD48a). This file provides approximate locations of 10300
 recombinations, an average of 52 per strain. Genotypes were generated
 using Affymetrix, MUGA, MegaMUGA, and GigaMUGA Illumina platforms.
 Microsatellites and eQTL genotypes were generated by the Williams/Lu
 laboratory. Unknown genotypes were imputed as B or D, or were called
 as H (heterozygous) if the genotype was uncertain. Genotypes were
 manually curated by RW Williams. Genotypes were smoothed to remove
 unlikely recombination events. Almost all recombinations are supported
 by multiple markers, although only one or two representative markers
 may be provided in this file. The original parent file
 (BXD\_El\_Grande\_Master\_Used\_to\_Proof\_Final\_Genotypes\_2016.xlxs)
 contains data for approximately 37000 markers. A subset of the most
 informative 7300 markers are included here. Genotypes for Chr Y and
 Chr M are provisional.

 The file does not include any markers for Chr Y or the mitochondrion.

 As of December 2018 GeneNetwork uses mm10 coordinates for mapping
 functions.

 (Updated Dec 19, 2018 by RW Williams)

 **2018-09-12: GeneNetwork 1 will be supported in 2018 but GeneNetwork
 2 will become the main production server**

 [GeneNetwork 2](http://gn2.genenetwork.org) is ready for testing and
 research projects. GeneNetwork 1 (GN1) and GeneNetwork 2 (GN2) share
 the a single database, but they use different code and algorithms for
 mapping and data analysis. There are some benefits of using GN2,
 including more powerful linear mixed model mapping methods that
 account for kinship structure.

 The interface of GN2 is still a work in progress. Please send any
 suggestion, requests, or bug reports to rwilliams@uthsc.edu.

 **2018-09-12: Notes on new BXD names**.

 1.  [BXD73](http://jaxmice.jax.org/strain/007117),
     [BXD73a](http://jaxmice.jax.org/strain/007124) (original known as
     *BXD80*), and [BXD73b](http://jaxmice.jax.org/strain/007146)
     (originally known as *BXD103*) are genetically very similar.
     [BXD73](http://jaxmice.jax.org/strain/007117) and *BXD80* are
     genetically identical at 82264 of 100290 markers (82% identical by
     descent). [BXD73](http://jaxmice.jax.org/strain/007117) keeps its
     original name and JAX identifier number (JR\#7117), whereas
     *BXD80* is now referred to as
     [BXD73a](http://jaxmice.jax.org/strain/007124) (JR\#7124).
     [BXD73](http://jaxmice.jax.org/strain/007117) and BXD103 are
     genetically identical at 90917 of 100290 markers (90.6% identical
     by descent). *BXD103* is now referred to as
     [BXD73b](http://jaxmice.jax.org/strain/007146) (JR\#7146).
 2.  [BXD48](http://jaxmice.jax.org/strain/007097) and
     [BXD48a](http://jaxmice.jax.org/strain/007139) (originally known
     as *BXD96*) are sister substrains, and are genetically identical
     at 93485 of 100290 markers (93.2% identical by descent).
     [BXD48](http://jaxmice.jax.org/strain/007097) retains its original
     name and JAX identifier number (JAX JR\#7097) whereas BXD96 is now
     referred to as [BXD48a](http://jaxmice.jax.org/strain/007139)
     (JR\#7139).
 3.  [BXD65](http://jaxmice.jax.org/strain/007110),
     [BXD65a](http://jaxmice.jax.org/strain/007140) (originally known
     as *BXD97*), and [BXD65b](http://jaxmice.jax.org/strain/009677)
     (originally known as *BXD92*) are sister substrains.
     [BXD65](http://jaxmice.jax.org/strain/007110) and *BXD97* are
     genetically identical at 92225 of 100290 markers (92% identical by
     descent). *BXD97* is now referred to as
     [BXD65a](http://jaxmice.jax.org/strain/007140) (JR\#7140).
     [BXD65](http://jaxmice.jax.org/strain/007110) and *BXD92* are
     genetically identical at 6155 of 6459 markers (95.3% identical by
     descent). [BXD65](http://jaxmice.jax.org/strain/007110) retains
     its original name and JAX identifier (JR\#7110) whereas *BXD92* is
     now referred to as [BXD65b](http://jaxmice.jax.org/strain/009677)
     (JR\#9677).

 **2018-09-08: UConn-Rheaume Retina RGC (Sep18) scRNA-Seq Log2 and raw
 data.** profiling by array entered into GeneNetwork.

 -   [UConn-Rheaume Retina RGC (Sep18) scRNA-Seq
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=866)
 -   [UConn-Rheaume Retina RGC (Sep18) scRNA-Seq
     Raw](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=865)
 -   [UConn-Rheaume Retina RGC (Sep18) scRNA-Seq
     Siamak](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=867)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-08-23: HSNIH-Palmer Orbitofrontal Cortex RNA-Seq (Aug18) rlog**
 profiling by array entered into GeneNetwork.

 -   [HSNIH-Palmer Orbitofrontal Cortex RNA-Seq (Aug18)
     rlog](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=864)
 -   [HSNIH-Palmer Prelimbic Cortex RNA-Seq (Aug18)
     rlog](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=863)
 -   [HSNIH-Palmer Lateral Habenula RNA-Seq (Aug18)
     rlog](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=862)
 -   [HSNIH-Palmer Infralimbic Cortex RNA-Seq (Aug18)
     rlog](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=861)
 -   [HSNIH-Palmer Nucleus Accumbens Core RNA-Seq (Aug18)
     rlog](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=860)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-08-14: EPFL/LISP BXD HFD+CD Liver Affy Mouse Gene 1.0 ST
 (Aug18) RMA** profiling by array entered into GeneNetwork.

 -   [EPFL/LISP BXD CD Liver Affy Mouse Gene 1.0 ST (Aug18)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=859)
 -   [EPFL/LISP BXD HFD Liver Affy Mouse Gene 1.0 ST (Aug18)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=858)
 -   [EPFL/LISP BXD CD+HFD Liver Affy Mouse Gene 1.0 ST (Aug18)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=857)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-08-10: NIA Aging BXD CD Liver RNA-Seq (Aug18) Log2** profiling
 by array entered into GeneNetwork.

 -   [NIA Aging BXD CD Liver RNA-Seq (Aug18)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=856)
 -   [NIA Aging BXD HFD Liver RNA-Seq (Aug18)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=855)
 -   [NIA Aging BXD CD+HFD Liver RNA-Seq (Aug18)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=854)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-08-07: NIA Aging BXD CD Liver Affy Clariom S Gene Level (Aug18)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [NIA Aging BXD CD Liver Affy Clariom S Gene Level (Aug18) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=853)
 -   [NIA Aging BXD HFD Liver Affy Clariom S Gene Level (Aug18) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=852)
 -   [NIA Aging BXD CD+HFD Liver Affy Clariom S Gene Level (Aug18) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=851)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-08-06: UTHSC BXD Liver Affy Clariom S GeneLevel Main (Aug18)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Liver Affy Clariom S GeneLevel Main (Aug18) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=850)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-04-27: JAX Retina (Apr18) RNA-Seq log2-Z** profiling by array
 entered into GeneNetwork.

 -   [JAX Retina (Apr18) RNA-Seq
     log2-Z](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=847)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-04-11: UCSD AIL Prefrontal Cortex (Apr18) RNA-Seq** profiling
 by array entered into GeneNetwork.

 -   [UCSD AIL Prefrontal Cortex (Apr18)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=846)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-04-11: UCSD AIL Striatum (Apr18) RNA-Seq** profiling by array
 entered into GeneNetwork.

 -   [UCSD AIL Striatum (Apr18)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=845)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-04-11: UCSD AIL Hippocampus (Apr18) RNA-Seq** profiling by
 array entered into GeneNetwork.

 -   [UCSD AIL Hippocampus (Apr18)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=844)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2018-01-04: UTHSC BXD Harvested Liver RNA-Seq (Aug18) Log2 \*\***
 profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Harvested Liver RNA-Seq (Aug18) Log2
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=843)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-09-29: UTHSC BXD Liver Affy Clariom S GeneLevel Main (Sep17)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Liver Affy Clariom S GeneLevel Main (Sep17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=842)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-09-12: EPFL/LISP BXD HFD Gastrointestinal Affy Clariom S
 GeneLevel Main (Sep17) RMA \*\*** profiling by array entered into
 GeneNetwork.

 -   [EPFL/LISP BXD HFD Gastrointestinal Affy Clariom S GeneLevel Main
     (Sep17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=841)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-09-12: EPFL/LISP BXD CD Gastrointestinal Affy Clariom S
 GeneLevel Main (Sep17) RMA \*\*** profiling by array entered into
 GeneNetwork.

 -   [EPFL/LISP BXD CD Gastrointestinal Affy Clariom S GeneLevel Main
     (Sep17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=840)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-09-12: EPFL/LISP BXD CD+HFD Gastrointestinal Affy Clariom S
 GeneLevel Main (Sep17) RMA \*\*** profiling by array entered into
 GeneNetwork.

 -   [EPFL/LISP BXD CD+HFD Gastrointestinal Affy Clariom S GeneLevel
     Main (Sep17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=839)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-08-24: UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction
 B, HFD Cohorts (Mar 2017) log2** profiling by array entered into
 GeneNetwork.

 -   [UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction B, HFD
     Cohorts (Mar 2017)
     log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=838)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-08-24: UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction
 B, Chow Diet Cohorts (Mar 2017) log2** profiling by array entered into
 GeneNetwork.

 -   [UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction B, Chow
     Diet Cohorts (Mar 2017)
     log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=837)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-08-24: UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction
 A, HFD Cohorts (Mar 2017) log2** profiling by array entered into
 GeneNetwork.

 -   [UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction A, HFD
     Cohorts (Mar 2017)
     log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=836)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-08-24: UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction
 A, Chow Diet Cohorts (Mar 2017) log2** profiling by array entered into
 GeneNetwork.

 -   [UTHSC/ETHZ/EPFL BXD Liver Polar Metabolites Extraction A, Chow
     Diet Cohorts (Mar 2017)
     log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=835)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-04-27: INIA-UTHSC Midbrain CIE Affy MTA 1.0 Exon Level (Apr17)
 RMA** profiling by array entered into GeneNetwork.

 -   [INIA-UTHSC Midbrain CIE Affy MTA 1.0 Exon Level (Apr17)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=834)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-05-05: INIA-UTHSC Striatum CIE Affy MTA 1.0 GeneLevel (Apr17)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [INIA-UTHSC Striatum CIE Affy MTA 1.0 GeneLevel (Apr17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=833)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-05-05: INIA-UTHSC Prefrontal Cortex CIE Affy MTA 1.0 GeneLevel
 (Apr17) RMA \*\*** profiling by array entered into GeneNetwork.

 -   [INIA-UTHSC Prefrontal Cortex CIE Affy MTA 1.0 GeneLevel (Apr17)
     RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=832)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-05-05: INIA-UTHSC Hippocampus CIE Affy MTA 1.0 GeneLevel
 (Mar18) RMA** profiling by array entered into GeneNetwork.

 -   [INIA-UTHSC Hippocampus CIE Affy MTA 1.0 GeneLevel (Mar18)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=831)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-04-28: INIA-UTHSC Midbrain CIE Affy MTA 1.0 GeneLevel (Apr17)
 RMA** profiling by array entered into GeneNetwork.

 -   [INIA-UTHSC Midbrain CIE Affy MTA 1.0 GeneLevel (Apr17)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=830)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-25: UTHSC Hippocampus Ethanol-Stress Affy MTA 1.0 Gene Level
 (Mar17) RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Hippocampus Ethanol-Stress Affy MTA 1.0 Gene Level (Mar17)
     RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=829)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-26: UTHSC Hippocampus Stress Affy MTA 1.0 Gene Level (Mar17)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Hippocampus Stress Affy MTA 1.0 Gene Level (Mar17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=828)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-27: UTHSC Hippocampus Ethanol Affy MTA 1.0 Gene Level
 (Mar17) RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Hippocampus Ethanol Affy MTA 1.0 Gene Level (Mar17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=827)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-28: UTHSC Hippocampus Saline Affy MTA 1.0 Gene Level (Mar17)
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Hippocampus Saline Affy MTA 1.0 Gene Level (Mar17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=826)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-29: UTHSC Hippocampus All Treatments Affy MTA 1.0 Gene Level
 (Mar17) RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Hippocampus All Treatments Affy MTA 1.0 Gene Level (Mar17)
     RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=825)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-24: UTHSC Human Islets Affy Human Gene 2.0 ST (Mar17) RMA
 \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC Human Islets Affy Human Gene 2.0 ST (Mar17) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=824)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA CXB Liver Affy M430 2.0 (Jan16) RMA** profiling by
 array entered into GeneNetwork.

 -   [UCLA CXB Liver Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=823)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA AXB/BXA Liver Affy M430 2.0 (Jan16) RMA** profiling
 by array entered into GeneNetwork.

 -   [UCLA AXB/BXA Liver Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=822)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA CXB Aorta Affy M430 2.0 (Jan16) RMA** profiling by
 array entered into GeneNetwork.

 -   [UCLA CXB Aorta Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=821)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA AXB/BXA Aorta Affy M430 2.0 (Jan16) RMA** profiling
 by array entered into GeneNetwork.

 -   [UCLA AXB/BXA Aorta Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=820)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA BXD Aorta Affy M430 2.0 (Jan16) RMA** profiling by
 array entered into GeneNetwork.

 -   [UCLA BXD Aorta Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=819)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-03-02: UCLA BXD Liver Affy M430 2.0 (Jan16) RMA** profiling by
 array entered into GeneNetwork.

 -   [UCLA BXD Liver Affy M430 2.0 (Jan16)
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=818)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-02-04: UCSD CFW Prefrontal Cortex (Jan17) RNA-Seq Log2
 Z-score** profiling by array entered into GeneNetwork.

 -   [UCSD CFW Prefrontal Cortex (Jan17) RNA-Seq Log2
     Z-score](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=813)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-02-04: UCSD CFW Striatum (Jan17) RNA-Seq Log2 Z-score**
 profiling by array entered into GeneNetwork.

 -   [UCSD CFW Striatum (Jan17) RNA-Seq Log2
     Z-score](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=812)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-02-04: UCSD CFW Hippocampus (Jan17) RNA-Seq Log2 Z-score**
 profiling by array entered into GeneNetwork.

 -   [UCSD CFW Hippocampus (Jan17) RNA-Seq Log2
     Z-score](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=811)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-01-17: UCSD CFW Prefrontal Cortex (Jan17) RNA-Seq** profiling
 by array entered into GeneNetwork.

 -   [UCSD CFW Prefrontal Cortex (Jan17)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=810)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-01-17: UCSD CFW Striatum (Jan17) RNA-Seq** profiling by array
 entered into GeneNetwork.

 -   [UCSD CFW Striatum (Jan17)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=809)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2017-01-17: UCSD CFW Hippocampus (Jan17) RNA-Seq** profiling by
 array entered into GeneNetwork.

 -   [UCSD CFW Hippocampus (Jan17)
     RNA-Seq](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=808)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-11-28: EPFL/LISP BXD CD Liver Affy Mouse Gene 1.0 ST (Apr13)
 RMA Exon Level** profiling by array entered into GeneNetwork.

 -   [EPFL/LISP BXD CD Liver Affy Mouse Gene 1.0 ST (Apr13) RMA Exon
     Level](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=849)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-11-28: EPFL/LISP BXD HFC Liver Affy Mouse Gene 1.0 ST (Apr13)
 RMA Exon Level** profiling by array entered into GeneNetwork.

 -   [EPFL/LISP BXD HFC Liver Affy Mouse Gene 1.0 ST (Apr13) RMA Exon
     Level](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=848)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-11-04: UTHSC B6D2 Retina Affy miRNA-4.0 (Nov16) RMA \*\***
 profiling by array entered into GeneNetwork.

 -   [UTHSC B6D2 Retina Affy miRNA-4.0 (Nov16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=806)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-10-02: BXD Genotypes file status (October 2016)**

 In September and October 2016, Robert Williams, Jesse Ingels, Lu Lu,
 and Danny Arends released new genotype files for many of the original
 BXD strains (BXD1 through BXD102), and for all of the new strains
 (BXD104 to BXD220). Genotypes were generated at about 74,000 SNPs
 using the GigaMUGA array developed by Drs. Fernando Pardo-Manuel de
 Villena (University of North Carolina) and Gary Churchill (The Jackson
 Laboratory). Genotypes were generated at GeneSeek (Neogen Inc) with
 financial support from the University of Tennessee Center for
 Integrative and Translational Genomics.

 The new genotypes are now available in GeneNetwork as the [2017
 Genotype
 file](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=600).
 All SNPs were mapped to the older July 2007 mm9 NCBI Build 37 assembly
 and to the newer Dec 2011, mm10, GRCm38 assembly. Of the 74,000
 GigaMUGA and many other markers we have typed, only about 6800 are
 useful (informative) in defining recombination events in the current
 set of BXDs. These informative markers either define unique
 recombination patterns across the 198 BXD strains (including extinct
 strains) or they define the proximal and distal ends of regions that
 do not contain any known recombinations in the BXD family.

 The file does not include any markers for Chr Y or the mitochondrion.

 As of October 2016 GeneNetwork still uses mm9 coordinates for mapping
 functions.

 (Updated Oct 2, 2016 by RW Williams)

 **2016-09-29: TIGEM Human Retina RNA-Seq (Sep16) RPKM log2** profiling
 by array entered into GeneNetwork.

 -   [TIGEM Human Retina RNA-Seq (Sep16) RPKM
     log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=802)

 This dataset is public available, please contact [Michele
 Pinelli](mailto:banfi@tigem.it) at [Telethon Institute of Genetics and
 Medicine (TIGEM)](http://www.tigem.it) for further information.
 (Implemented by Michele Pinelli, Arthur Centeno and Rob Williams).

 **2016-09-09: UTHSC B6D2 Retina Affy MoGene 1.0ST (Sep16) Exon Level
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC B6D2 Retina Affy MoGene 1.0ST (Sep16) Exon Level RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=804)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-09-09: UTHSC B6D2 Retina Affy MoGene 1.0ST (Sep16) Gene Level
 RMA \*\*** profiling by array entered into GeneNetwork.

 -   [UTHSC B6D2 Retina Affy MoGene 1.0ST (Sep16) Gene Level RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=803)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2016-07-26: Raw and processed data from our Hippocampus studies
 based on publication "Genetics of the hippocampal transcriptome in
 mouse: a systematic survey and online neurogenomics resource" can be
 accessed now through GEO series GSE84767.**

 -   [Data Set Group: Hippocampus Consortium
     M430v2](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84767)
     Pubmed ID:[20582282](http://www.ncbi.nlm.nih.gov/pubmed/20582282)
 -   have been assigned with a [GEO Series record
     GSE84767](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84767)
     which provides access to all of the data and is the accession that
     should be quoted in any manuscript discussing the data.

 **2016-07-16: Gene expression data for 39 inbred mice strains for CD4+
 T cells** profiling by array entered into GeneNetwork.

 -   [HMS Spleen CD4+ T cells MDP Affy Mouse Gene 1.0 ST (Jul16) Gene
     Level](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=801)
 -   [HMS Spleen CD4+ T cells MDP Affy Mouse Gene 1.0 ST (Jul16) Exon
     Level](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=804)

 [GEO Series:
 GSE60337](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60337).
 To determine the breadth and underpinning of changes in immunocyte
 gene expression due to genetic variation in mice, we performed, as
 part of the Immunological Genome Project, gene expression profiling
 for CD4(+) T cells and neutrophils purified from 39 inbred strains of
 the Mouse Phenome Database.

 **2016-05-16: FGUCAS BXH/HXB Brown Adipose Affy Rat Gene 2.0 ST
 (May16) log2** profiling by array entered into GeneNetwork.

 -   [FGUCAS BXH/HXB Brown Adipose Affy Rat Gene 2.0 ST (May16) log2
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=799)

 Currently this dataset is confidential, please contact [Michal
 Pravenec](mailto:pravenec%20@%20biomed.cas.cz) Czech Academy of
 Sciences for further information. (Implemented by Michal Pravenec,
 Arthur Centeno and Rob Williams).

 **2016-04-16: EPFL/ETHZ BXD Brown Adipose Proteome (Apr16)** profiling
 by array entered into GeneNetwork.

 -   [EPFL/ETHZ BXD Brown Adipose, Isolated Mitochondria Proteome, Chow
     Diet (Apr16) Light SWATH
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=798)
 -   [EPFL/ETHZ BXD Brown Adipose, Total Tissue Proteome, Chow Diet
     (Apr16) Light SWATH
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=797)

 Currently this dataset is confidential, please contact [Johan
 Auwerx](mailto:admin.auwerx@epfl.ch) Ecole Polytechnique Federale de
 Lausanne for further information. (Implemented by Johan Auwerx, Arthur
 Centeno and Rob Williams).

 **2016-04-16: DoD TATRC Retina Affy MoGene 2.0 ST** profiling by array
 entered into GeneNetwork.

 -   [DoD Retina Blast vs Normal Retina Affy MoGene 2.0 ST (Apr16) RMA
     Exon Level
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=796)
 -   [DoD Retina Blast vs Normal Retina Affy MoGene 2.0 ST (Apr16) RMA
     Gene Level
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=787)
 -   [DoD Retina After Blast Affy MoGene 2.0 ST (Mar16) RMA Exon Level
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=786)
 -   [DoD Retina After Blast Affy MoGene 2.0 ST (Mar16) RMA Gene Level
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=785)

 Currently this dataset is confidential, please contact [Eldon
 Geisert](mailto:egeiser@emory.edu) Emory Eye Center for further
 information. (Implemented by Eldon Geisert, Arthur Centeno and Rob
 Williams).

 **2016-04-16: VCU BXD NAc EtOH vs CIE Air M430 2.0** profiling by
 array entered into GeneNetwork.

 -   [VCU BXD NAc CIE EtOH Masked ComBat M430 2.0 (Apr16) RMA Z-score
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=795)
 -   [VCU BXD NAc CIE EtOH Masked ComBat M430 2.0 (Apr16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=794)
 -   [VCU BXD NAc CIE Air Masked ComBat M430 2.0 (Apr16) RMA Z-score
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=793)
 -   [VCU BXD NAc CIE Air Masked ComBat M430 2.0 (Apr16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=792)
 -   [VCU BXD PFC CIE EtOH Masked ComBat M430 2.0 (Apr16) RMA Z-score
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=791)
 -   [VCU BXD PFC CIE EtOH Masked ComBat M430 2.0 (Apr16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=790)
 -   [VCU BXD PFC CIE Air Masked ComBat M430 2.0 (Apr16) RMA Z-score
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=789)
 -   [VCU BXD PFC CIE Air Masked ComBat M430 2.0 (Apr16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=788)

 Currently this dataset is confidential, please contact [Michael
 Miles](mailto:mfmiles@vcu.edu) Virginia Commonwealth University for
 further information. (Implemented by Michael Miles, Arthur Centeno and
 Rob Williams).

 **2016-02-16: UCAMC LXS Whole Brain Saline and Ethanol RNA Sequence
 FPKM** profiling by array entered into GeneNetwork.

 -   [UCAMC LXS Whole Brain Saline RNA Sequence (Feb16) FPKM
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=784)
 -   [UCAMC LXS Whole Brain Ethanol RNA Sequence (Feb16) FPKM
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=784)

 Currently this dataset is confidential, please contact [Richard
 Radcliffe](mailto:richard.radcliffe@ucdenver.edu) University of
 Colorado at Denver for further information. (Implemented by Richard
 Radcliffe, Arthur Centeno and Rob Williams).

 **2016-02-16: UTHSC Hippocampus Illumina v6.1 NOS, NOE, RSS, and RSE
 data sets** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Hippocampus ILM v6.1 RSS Balanced (Feb16)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=782)
 -   [UTHSC BXD Hippocampus ILM v6.1 RSE Balanced (Feb16)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=781)
 -   [UTHSC BXD Hippocampus ILM v6.1 NOE Balanced (Feb16)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=780)

 This is untreated control "Base" group gene expression data for the
 hippocampus of BXD strains of mice (n = 27 strains and n = 35
 animals). These data NON data are useful as baseline for comparison
 with NOS, NOE, RSS, and RSE data sets. NON = NON = No stress and no
 saline control injection; NOS = No restraint stress and given only
 saline injections prior to sacrifice; NOE = No restraint stress and
 given an ethanol injection prior to sacrifice; RSS = short restraint
 stress (1 episode) followed by a saline injection; and finally, RSE =
 Restraint stress followed by an ethanol injection.

 For more details on the precise experimental paradigm, please see
 [Ziebarth et al
 2010](http://www.nervenet.org/papers/Ziebarth-EtOH.pdf) or the
 original paper that used this paradigm by Kerns RT, Ravindranathan A,
 Hassan S, Cage MP, York T, Williams RW, Miles MF
 ([2005](http://www.jneurosci.org/cgi/reprint/25/9/2255))
 Ethanol-responsive brain region expression networks: implications for
 behavioral responses to acute ethanol in DBA/2J versus C57BL/6J mice.
 Journal of Neuroscience 25: 2255-2266.

 Please contact [Rob Williams](mailto:rwilliams@uthsc.edu) University
 of Tennessee Health Science Center for further information.
 (Implemented by Lu Lu, Arthur Centeno and Rob Williams).

 **2016-02-16: EPFL/LISP BXD CD+HFD Subcutaneous WAT Affy MTA 1.0
 GeneLevel Main (Feb16) RMA** profiling by array entered into
 GeneNetwork.

 -   [EPFL/LISP BXD CD Subcutaneous WAT Affy MTA 1.0 Gene Level Main
     (Feb16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=779)
 -   [EPFL/LISP BXD HFD Subcutaneous WAT Affy MTA 1.0 Gene Level Main
     (Feb16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=778)
 -   [EPFL/LISP BXD CD+HFD Subcutaneous WAT Affy MTA 1.0 Gene Level
     Main (Feb16) RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=777)

 The BXD genetic reference population is a recombinant inbred panel
 descended from crosses between the C57BL/6 (B6) and DBA/2 (D2) strains
 of mice, which segregate for about 5 million sequence variants.
 Recently, some these variants have been established with effects on
 general metabolic phenotypes such as glucose response and bone
 strength. In this study, we examined both genetic variants across 40
 strains of BXD and the two founder lines, in addition to a major
 environmental influence—long term feeding with chow diet (CD) or high
 fat diet (HFD)—to see how metabolic gene expression varies by genotype
 and environment, and gene-by-environment interactions. The basic heart
 phenotypes quantified in these cohorts were not affected by HFD
 feeding (e.g. blood pressure and heart rate).

 Currently this dataset is confidential, please contact [Johan
 Auwerx](mailto:admin.auwerx@epfl.ch) Ecole Polytechnique Federale de
 Lausanne for further information. (Implemented by Johan Auwerx, Arthur
 Centeno and Rob Williams).

 **2015-12-09: INIA LCM (11 Regions) RNA-seq Transcript Level (Dec15)**
 profiling by array entered into GeneNetwork.

 -   [INIA LCM (11 Regions) CIE/AIR/BAS RNA-seq Transcript Level
     (Dec15)](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=776)
 -   [INIA LCM (11 Regions) BASELINE RNA-seq Transcript Level
     (Dec15)](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=775)
 -   [INIA LCM (11 Regions) CIE/AIR RNA-seq Transcript Level
     (Dec15)](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=772)

 This dataset is now public, please contact [Megan
 Mulligan](mailto:mmulliga@uthsc.edu) University of Tennessee Health
 Science Center for further information. (Implemented by Megan
 Mulligan, Arthur Centeno and Rob Williams).

 **2015-12-07: UTHSC Affy MoGene 1.0 ST Spleen** profiling by array
 entered into GeneNetwork.

 -   [UTHSC Affy MoGene 1.0 ST Spleen (Dec10) RMA
     Set2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=774)
 -   [UTHSC Affy MoGene 1.0 ST Spleen (Dec10) RMA
     Set1](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=773)

 This is a preliminary release with known errors of a spleen gene
 expression data set generated by a DOD-funded consortium (Byrne, Kotb,
 Williams, and Lu). Please contact Lu Lu or Robert Williams regarding
 status of this data set. Animals were generated at UTHSC by Lu Lu and
 colleagues. The spleen of untreated young adult mice was profiled
 using the Affymetrix GeneChip Mouse Gene 1.0 ST array that contains
 approximately 34,728 probe sets that target approximately 29,000 well
 defined transcripts (RefSeq mRNA isoforms) and essentially all known
 protein coding genes in mouse. This array is an "exon style" array
 with multiple probes in all known exons of each gene (an average of
 about 27 per gene) and is an abridged version of the Affymetrix Exon
 1.0 ST array. However, it also does contain some probes that target
 non-coding RNAs and even miRNA precursors (search "ncrna").This
 dataset is now public, please contact [Robert
 Williams](mailto:rwilliams@uthsc.edu) University of Tennessee Health
 Science Center for further information. (Implemented by Lu Lu, Arthur
 Centeno and Rob Williams).

 **2015-11-10: Hippocampus Mouse Transcriptome Assay 1.0 GeneLevel Main
 (Nov15) RMA** profiling by array entered into GeneNetwork.

 -   [Hippocampus Mouse Transcriptome Assay 1.0 Gene Level Main (Nov15)
     RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=771)

 Please contact [Megan Mulligan](mailto:mmulliga@uthsc.edu) University
 of Tennessee Health Science Center for further information.
 (Implemented by Megan Mulligan, Arthur Centeno and Rob Williams).

 **2015-10-13: UTHSC BXD Hippocampus ILM v6.1 RSE Balanced (Feb17)
 RankInv** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Hippocampus ILM v6.1 RSE Balanced (Feb17)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=817)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2015-10-14: UTHSC BXD Hippocampus ILM v6.1 RSS Balanced (Feb17)
 RankInv** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Hippocampus ILM v6.1 RSS Balanced (Feb17)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=816)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2015-10-15: UTHSC BXD Hippocampus ILM v6.1 NOE Balanced (Feb17)
 RankInv** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Hippocampus ILM v6.1 NOE Balanced (Feb17)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=815)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2015-10-16: UTHSC BXD Hippocampus ILM v6.1 NOS Balanced (Feb17)
 RankInv** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Hippocampus ILM v6.1 NOS Balanced (Feb17)
     RankInv](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=814)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

 **2015-09-14: The Genotype-Tissue Expression (GTEx)** is providing a
 comprehensive atlas of gene expression and regulation across 53 human
 tissues. The latest GTEx data release (**dbGaP release
 phs000424.v5.p1**) is now available on GeneNetwork.

 -   [All 53 Tissues, RNA-Seq GTEx dbGaP release
     phs000424.v5.p1](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=770)

 Note: Please disregard confusing legend provided in Study tab of the
 study page which claims that study contains 552 subjects with
 genotypes - those totals are counts of subjects with all molecular
 data types, not just molecular genotypes.

 More detailed representation of subject counts in molecular datasets
 (including genotypes) may be found in 'Molecular Data' tab at [Common
 Fund (CF) Genotype-Tissue Expression Project (GTEx) dbGaP Study
 Accession:
 phs000424.v5.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/molecular.cgi?study_id=phs000424.v5.p1).

 **2015-05-30: UTHSC BXD Aged Hippocampus Affy Mouse Gene 1.0 ST
 (Jun15) Exon Level RMA** profiling by array entered into GeneNetwork.

 -   [UTHSC BXD Aged Hippocampus Affy Mouse Gene 1.0 ST (Jun15) Exon
     Level
     RMA](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=805)

 This dataset is publicly available in most cases. For more information
 on how to access the data please refer to the link above.

* * * * *

[See More News](#)\

  **2014-10-21: HZI Lung Flu Infected BXD (Nov16) RNA-Seq \*\***
  profiling by array entered into GeneNetwork.
 
  -   [HZI Lung Flu Infected BXD (Nov16) RNA-Seq
      \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=807)
 
  This dataset is publicly available in most cases. For more
  information on how to access the data please refer to the link
  above.

 **2014-10-10: BXD Elicited Peritoneal Neutrophils Gene Expression**
 profiling by array entered into GeneNetwork.

 -   [UTHSC Elicited Peritoneal Neutrophils Affy MoGene 2.0 ST (Oct14)
     RMA
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=595)
 -   [UTHSC Elicited Peritoneal Neutrophils Affy MoGene 2.0 ST (Oct14)
     RMA Exon Level
     \*\*](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=596)

 Currently this dataset is confidential, please contact [Marko
 Radic](mailto:mradic@uthsc.edu) University of Tennessee Health Science
 Center for further information. (Implemented by Marko Radic, Indira
 Neeli, Teruki Hagiwara, Arthur Centeno and Rob Williams).

 **2014-08-29: Mouse Diversity Panel Hippocampus Antidepressant**
 profiling by array entered into GeneNetwork.

 -   UFL MDP Hippocampus Antidepressant Affy Mouse 430 2.0 (Aug14) RMA
     \*\*

 Currently this dataset is confidential, please contact [Brooke
 Miller](mailto:brookemiller@ufl.edu) University of Florida for further
 information. (Implemented by Brooke Miller, Arthur Centeno and Rob
 Williams).

 **2014-08-10: RTI RCMRC BXD Fecal Metabolites CD+HFD (Aug14) Log2**
 profiling by array entered into GeneNetwork.

 -   [RTI RCMRC BXD Fecal Metabolites CD+HFD (Aug14)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=717)
 -   [RTI RCMRC BXD Fecal Metabolites HFD (Aug14)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=716)
 -   [RTI RCMRC BXD Fecal Metabolites CD (Aug14)
     Log2](http://genenetwork.org/webqtl/main.py?FormID=sharinginfo&GN_AccessionId=715)

 The goal of this project is to study expression of a large set of
 defined and undefined metabolites in fecal samples from a genetically
 diverse set of BXD mouse strains (females) raised either on a high fat
 diet or low fat diet. Samples were taken from animals at sacrifices
 directly from the large intestine.

 Aim: Effect of genetics and diet on fecal metabolomics\
  Use UPLC-MS Broad Spectrum Metabolomics to study:Mice fecal samples
 To capture the most signal, all samples were analyzed under positive
 and negative ion mode using a reversed-phase separation. Please
 contact [Robert Clark](mailto:rclark@rti.org) RTI International for
 further information. (Implemented by Robert Clark, Arthur Centeno and
 Rob Williams).

 **2014-05-02: BXD Liver, Soluble Proteins CD and Hi-Fat Gene
 Expression** profiling by array entered into GeneNetwork.

 -   EPFL/ETHZ BXD Liver, Soluble Proteins CD (May14) SWATH \*\*
 -   EPFL/ETHZ BXD Liver, Soluble Proteins HFD (May14) SWATH \*\*

 Currently this dataset is confidential, please contact [Johan
 Auwerx](mailto:admin.auwerx@epfl.ch) Ecole Polytechnique Federale de
 Lausanne for further information. (Implemented by Johan Auwerx, Evan
 Williams, Arthur Centeno and Rob Williams).

 **2014-04-03: UTHSC Mouse BXD Gastrointestinal Affy MoGene 1.0 ST Gene
 Level (Apr14) RMA** profiling by array entered into GeneNetwork.

 -   UTHSC Mouse BXD Gastrointestinal Affy MoGene 1.0 ST Gene Level
     (Apr14) RMA \*\*

 Currently this dataset is confidential, please contact [Dennis D
 Black](mailto:dblack@uthsc.edu) UT Le Bonheur Pediatric Specialists
 for further information. (Implemented by Dennis D Black, Arthur
 Centeno and Rob Williams).

 **2014-03-07: The Genotype-Tissue Expression (GTEx)** is providing a
 comprehensive atlas of gene expression and regulation across multiple
 human tissues. The latest GTEx data release (dbGaP release
 phs000424.v4.p1) is now available on
