<P>
The BXD family of recombinant inbred (<A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">RI</A>) strains were derived by crossing C57BL/6J (B6) and DBA/2J (D2) and inbreeding progeny for 20 or more generations. This genetic reference panel is a remarkable resource because data for thousands of phenotypes and nearly 100 gene, protein, and metabolite expression data sets have been acquired over a nearly a 40-year period. Another advantage of the BXD family is that the both parents have been sequenced (C57BL/6J as part of a public effort, and DBA/2J by Celera Genomics, by the UTHSC group and by Sanger). Based on our analysis of the sequence data, these two strains differ at approximately 4.8 million SNPs. Variants (mostly single nucleotide polymorphisms and about 500,000 insertion-deletions) that produce interesting phenotypes can be located efficiently. The zoomable physical maps in GeneNetwork can display the positions of B versus D-type SNPs at high resolution.
</P>

Our DBA/2J sequence data (from Wang et al. 2016) have been used to generate a virtual genome for this strain using a C57BL/6J framework. In other words, all SNPs and small DBA/2J indels have inserted in place of original C57BL/6J sequence. 



<P>
<DIR>
<A HREF="http://gn1.genenetwork.org/bxd/"><img src="/images/upload/BXD20.png" alt="BXD20" border= 0 valign="middle" width="273" height="178"></A>
</DIR>
<DIR><P><SMALL><B>Legend:</B> Photo gallery of BXD strains</SMALL></P>
</DIR>



<P><B>EPOCH DIFFERENCES or "Batch Effect" among BXD strains</B>. BXD strains (1 through 103) were produced as at least four separate groups or subfamilies. BXD1 through BXD30 were produced by Benjamin A. Taylor starting in about 1971, with the first publication using early generation BXD lines at F7 to F10 in 1973 (<A HREf="http://www.ncbi.nlm.nih.gov/pubmed/4719448">Taylor et al., 1973</A>  <A HREF="/images/upload/Taylor_1973.pdf"> Full text</A>, 1975 (<A HREf="http://www.ncbi.nlm.nih.gov/pubmed/807855">Taylor et al., 1975</A>, <A HREf="http://www.ncbi.nlm.nih.gov/pubmed/1203051">Womack et al., 1975</A>). A distinction is made between an RI line, which is not necessarily fully inbred (<20 F generations of inbreeding, and an RI strain, which should be the progeny of 20 or more sequential sib matings). 

<P>BXD31 and BXD32 are exceptional BXD strains. They were created by the Mouse Mutant Resource at The Jackson Laboratory in the propagation of different visible mutations.  Mutations arose in C57BL/6J or DBA/2 and then outcrossed to the other strain and subsequently inbred while maintaining the mutation. In the case of BXD31, inbreeding was followed after the F2 generation. In BXD32, there was a backcross to DBA/2 before sib mating. BXD32 is therefore approximately 75% DBA/2 and 25% B6. When Taylor learned about these strains he decided that they would be a useful addition to the BXD RI set. He eliminated the visible mutations, continued inbreeding, and added them to the BXDs set. The first publication using these two strains was in 1980. Both are usually lumped together with BXD1 through BXD30 as a "single" first cohort, but obviously, they should perhaps be excluded from all cohorts or BXD "epochs." BXD32 is also exceptional because it has a <I>D</I> mitochondrial genome and a <I>B</I>-type Y chromosome. (Information in this paragraph mainly from BA Taylor, email of Aug 17, 2014 to RWW).

<P>BXD33 through BXD42 were also produced by Benjamin Taylor (Taylor et al. <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/10087289">1999</A>), but from a new set of F2 crosses initiated in the early 1990s.

<P>BXD43 through BXD102 were produced by Lu Lu, Jeremy Peirce, Lee M. Silver, and Robert W. Williams in the late 1990s and early 2000s using advanced intercross progeny (Peirce et al. <A href="http://www.biomedcentral.com/1471-2156/5/7" target="_blank" class="fs14">2004</A>). These strains have roughly twice the number of recombinations as conventional F2-derived RI strains.

<P>BXD104 through BXD157 were produced by Lu Lu and Robert W. Williams starting in 2008 using standard F2 stock. 

<P>BXD160 through BXD186 were produced by Lu Lu and Robert W. Williams starting in 2010 using G8 and G9 advanced intercross stock donated by Abraham Palmer. 

<P>BXD187 through BXD220 were produced by Lu Lu and Robert W. Williams starting in 2014 using F2 stock. 

<P> Initial genotypes in 2008 highlighted breeding errors that resulted in sets of very closely related sister substrains that are almost genetically identical. In collaboration with the Jackson Laboratory, we have made the following changes in strain names to clarify the strong genetic relations among the newer BXD strains. 

<P>In general, data for the following sister substrains needs to be handled with special care prior to or during statistical analysis or gene mapping for the simple reason that the substrains are not genetically independent. However, if investigators do discover significant phenotype differences among strains, then data can be treated independently.  (The latest version of GeneNetwork (GN 2) includes a mapping method called Python Linear Mixed Model (pyLMM) that corrects for the effects of shared pedigrees and corrects mapping results (code written by Nick Furlotte in 2011-2013). )

<OL>

<LI> <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a>, <a href="http://jaxmice.jax.org/strain/007124" target="_blank">BXD73a</a> (original known as BXD80), and <a href="http://jaxmice.jax.org/strain/007146" target="_blank">BXD73b</a> (originally known as BXD103) are genetically very similar. <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> and BXD80 are genetically identical at 82264 of 100290 markers (82% identical by descent). <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> keeps its original name and JAX identifier number (JR#7117), whereas BXD80 is now referred to as <a href="http://jaxmice.jax.org/strain/007124" target="_blank">BXD73a</a> (JR#7124). <a href="http://jaxmice.jax.org/strain/007117" target="_blank">BXD73</a> and BXD103 are genetically identical at 90917 of 100290 markers (90.6% identical by descent). BXD103 is now referred to as <a href="http://jaxmice.jax.org/strain/007146" target="_blank">BXD73b</a> (JR#7146). 

<LI> <a href="http://jaxmice.jax.org/strain/007097" target="_blank">BXD48</a> and <a href="http://jaxmice.jax.org/strain/007139" target="_blank">BXD48a</a> (originally known as BXD96) are sister substrains, and are genetically identical at 93485 of 100290 markers (93.2% identical by descent). <a href="http://jaxmice.jax.org/strain/007097" target="_blank">BXD48</a> retains its original name and JAX identifier number (JAX JR#7097) whereas BXD96 is now referred to as <a href="http://jaxmice.jax.org/strain/007139" target="_blank">BXD48a</a> (JR#7139). 

<LI> <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a>, <a href="http://jaxmice.jax.org/strain/007140" target="_blank">BXD65a</a> (originally known as BXD97), and <a href="http://jaxmice.jax.org/strain/009677" target="_blank">BXD65b</a> (originally known as BXD92) are sister substrains.  <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> and BXD97 are genetically identical at 92225 of 100290 markers (92% identical by descent). BXD97 is now referred to as <a href="http://jaxmice.jax.org/strain/007140" target="_blank">BXD65a</a> (JR#7140). <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> and BXD92 are genetically identical at 6155 of 6459 markers (95.3% identical by descent). <a href="http://jaxmice.jax.org/strain/007110" target="_blank">BXD65</a> retains its original name and JAX identifier (JR#7110) whereas BXD92 is now referred to as <a href="http://jaxmice.jax.org/strain/009677" target="_blank">BXD65b</a> (JR#9677).

</OL>

<P> 

<P>While the strains used to generate these subsets of BXDs have the same official names and were all made using stock from the Jackson Laboratory, the individual parents were are not genetically identical due to inevitable genetic drift and mutation. Shifman and colleagues detected a surprisingly large number of new SNPs (n = 47 out of about 13000 SNPs studied) in the set of strains generated by BA Taylor in the early 1990s, and a small number (n = 5) of even newer SNPs in the set of BXD strains generated at UTHSC in the late 1990s (see <A HREF="http://http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.0040395">Shifman et al., 2006</A>).

<DIR><SMALL><P>"In the BXD set, 52 SNPs showed variation in genotypes that corresponded to the different phases of development of the BXD RIs [24–26] (Table S4). Forty-seven SNPs are not polymorphic in the 26 BXD strains established from a single cross of a C57BL/6J female to a DBA/2J male, but are polymorphic in similar BXD strains established more than 20 y later. Five SNPs are not polymorphic in the first 36 BXD strains, but are polymorphic in the newest set of 53 BXD lines (BXD43–100)."
</DIR></SMALL>

<P><B>Correction for Family or Epoch Substructure</B>

<P>The BXDs have the following epoch substructure:
<OL>
<LI> BXD1 through 30 make up the first epoch. Breeding for this group of BXD strains started in about 1970, with the first publication of fully inbred BXD strains in 1975 (Taylor et al., <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/807855>1975</A>)see Trait ID 13032 from BXD Published Phenotypes).
 
<LI>BXD31 and 32 are usually considered part of this same epoch but were not part of the same original cross, and their first use in a publication did not occur until the early 1980 (e.g., Brownstein, <A HREF="http://iai.asm.org/cgi/reprint/41/1/308?view=long&pmid=6305840>1983</A>, see Trait ID 10715). In fact, BXD32 has a mitochondrion that is inherited from DBA/2J. BXD32 could be considered the first DXB strain (DXB32).

<LI>BXD33 to BXD42 make up Ben Taylor's final addition to the BXD strains (Taylor et al, <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/10087289>1999</A>). These strains were added to enhance the utility of the BXDs for complex trait analysis starting in about 1992. The first published use of this strains appears was in 2001 (Williams et al., <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/11529276?dopt=Abstract>2001</A>, Trait ID 10645).

<LI>BXD43 to BXD103. This is a complex cohort of strains generated at UTHSC from advanced intercross progeny (Peirce et al., <A HREF="http://www.biomedcentral.com/1471-2156/5/7">2004</A>). 

<LI>BXD104 to BXD157. This is a single cohort of strains generated at UTHSC from F2 intercross stock. 

<LI>BXD160 to BXD186. This is a single cohort of strains generated at UTHSC from G8 and G9 advanced intercross progeny donated to RW Williams by Abraham Palmer in 2008. 

<LI>BXD187 to BXD220. This is a single cohort of strains generated at UTHSC from F2 intercross stock. 

</OL>

Users of the expanded BXD panel should take this epoch substructure into account. This is easy to do using the "Epoch" traits that are included in the BXD Phenotype database. For example, BXD Phenotype 12688 (BXD epoch batch trait 1) provides a simple code for the major phases of BXD production using the code of -1 for the first set through to BXD32, 0 for the second set (33 to 42), and +1 for the newer UTHSC set (43 to 103). 

<OL>
<LI> Determine whether your trait covaries well with any one of the three Epoch traits in GeneNetwork. Also check the status of BXD31 and BXD32. They may not belong to any group.

<LI>Determine if your trait maps extremely well to Chr 4 at 62 Mb (near the ALAD segmental duplication in DBA/2J).

</OL> 

<P>Strain nomenclature:  Some of the BXD strains have accumulated new mutations that have recently been characterized. When these mutations are known, the full nomenclature of the strain is now being modified. For example, BXD24/TyJ (aka BXD24 in most GeneNetwork databases), suffered a mutation in the <I>Cep290</I> gene in the late 1980s. The mutant allele (<I>rd16</I> is associated with autosomal recessive retinal degeneration. The original BXD strain was briefly referred to as BXD24a/TyJ, while the blind co-isogenic mutant was referred to as BXD24b/TyJ. The great majority of phenotype, expression, and genotype data in GeneNetwork was generated using these blind BXD24b/TyJ animals. However, in 2010, the nomenclature was changed again and the blind variant (JAX stock 000031) is now known as  BXD24/TyJ-Cep290rd16/J. The original BXD with normal vision was rederived from frozen stock and is now known once again as BXD24/TyJ, although the stock number has now been changed to 005243.

<P>BXD29/TyJ was also known as BXD29/TyJ-Tlr4, but is now formally BXD29-Tlr4lps-2J/J (JAX stock 000029). The original non-mutant stock is currently known as BXD29/TyJ again but the stock number of these rederived non-mutants has been changed to 010981.

<P>The mitochondrial DNA of all BXD strains were typed by Jing Gu and Shuhua Qi (Nov 2004) using DNAs obtained from the Jackson Laboratory (BXD1 through 42) or from the UTHSC colony. This typing relied on a SNP marker identified by Jan Jiao in Weikuan Gu's laboratory at nucleotide position 9461 in the reference C57BL/6J mitochondrial sequence. Most strains have inherited mitochondria from C57BL/6J. However, the following strains have mitochondria with a <I>D</I> allele at the rs8281487 (UT-M-9461) SNP in ND3 (T is reference and C is alternate allele): BXD32, 61, 74, 78, 82, 89, 90, 91, 95, and BXD99. These ten strains could be considered DXB recombinant inbred strains. </P>

<P>Genotypes of these strains: All BXD strains were genotyped in the first half of 2005 at 13377 markers as part of a CTC-Wellcome Trust collaboration. When combined with previous markers, there are a total of 7636 informative markers that differ betweeen the parental strains and that are useful for mapping with the BXD strains. The locations of these makers are known on the latest assembly of the mouse genome (Build 34, mm6). The median distance between these informative markers is 178,831 bp. The mean distance is 324,493 bp. There are only 26 intervals between markers that are longer than 5 Mb. No interval is greater than 10 Mb except on Chr X. These long intervals are essentially monomorphic between the parental strains. 
</P>

<P>The BXD genotype file used in GeneNetwork up to 2014 includes a selected subset of approximately 3795 markers (out of 7636) and includes all those markers with unique strain distribution patterns (<A HREF="http://gn1.genenetwork.org/glossary.html#SDP" target="_blank" class="fs14">SDP</A>) as well as pairs of markers--the most proximal and most distal--for SDPs represented by two or more markers. Slightly updated versions of this BXD genotype data set can be downloaded by ftp at <A HREF="ftp://atlas.uthsc.edu/Public/BXD_WebQTL_Genotypes" target="_blank" class="fs14">ftp://atlas.uthsc.edu/Public/BXD_WebQTL_Genotypes</A>.
</P>

<P>There are a total of 1848 known recombinations in the 36 older (JAX) BXD set; an average of 48.1 recombinations per strain. </P>

<P>There are a total of 4366 known recombinations in the 53 of the first set of UTHSC BXD strains (BXD43 to BXD102); an average of 82.4 recombinations per strain (Shifman et al., 2006). These RI strains were generated from an advanced intercross, and this accounts for the higher recombination load (Peirce et al., 2005).</P>

<P>The "classic" genotypes of the BXD strains (used through Dec 2016 in GeneNetwork) rely mainly on the Mouse Universal Genotyping Array (MUGA) genotyping platform. They also rely to some extent and earlier genotyping in 2008 using the Affymetrix Mouse Diversity array and even earlier work by Williams et al (2001) using microsatellite markers.

<P>In January 2017, the genotypes of most extant BXD strains were updated. The new data also include initial genotypes for the newest cohort of BXDs (BXD104 to BXD220). This January 2017 genotype file provides consensus genotypes for 198 BXD strains. Of the 198 BXD strains, 191 are independent, whereas 7 are substrains (e.g., BXD48 and BXD48a). This file provides approximate locations of 11500 recombinations, an average of 58 per strain. Genotypes were generated using Affymetrix, MUGA, MegaMUGA, and GigaMUGA Illumina platforms. Microsatellites and eQTL genotypes were generated by the Williams and Lu laboratory. Unknown genotypes were imputed as B or D, or were called as H (heterozygous) if the genotype was uncertain. Genotypes were manually curated by RW Williams. Genotypes were smoothed to remove unlikely recombination events. Almost all recombinations are supported by multiple markers, although only one or two representative markers may be provided in this file. The original parent file (BXD_El_Grande_Master_Used_to_Proof_Final_Genotypes_2016.xlxs) contains data for approximately 37000 markers. A subset of the most informative 7300 markers are included in the 2017 genotype file. Genotypes for Chr Y and  Chr M are provisional and will be verified in 2017. As of 2016, many strains with higher numbers (BXD100 and above) are not fully inbred.


<P>Approximately 5100 phenotypes are currently included in the BXD Phenotype database in GeneNetwork as of Jan 2017. You can get an update on this number by typing in an asterisk (*) in the search box. </P>

<P>How to obtain these strains: Please see <a href="http://jaxmice.jax.org/strain/000105.html"  target="_blank" class="fs14">http://jaxmice.jax.org/strain/000105.html</a>. Cost of the JAX BXD strains was approximately $65.40 each in 2008, $135.00 each in 2014, and $139.90 in 2017. To obtain strains BXD43 and higher please contact <A HREF="mailto:rwilliams@uthsc.edu" class="fs14">Rob Williams</A>. All strains through BXD102 are now fully inbred. We expect to generate as many as 160 viable BXDs strains by 2020. All extant BXD strains are being sequenced in early 2017 at 30X using 10X Chromium libraries at Hudson Alpha (supported by the UTHSC CITG, Williams, Lu, and colleagues at UTHSC, Abraham Palmer and colleagues at UCSD, and by Jonathan Pritchard at Stanford).
</P>

<P>For more details on the history, generation, and use of RI strains as genetic reference populations for systems genetics please see Silver <A HREF="http://www.complextrait.org/archive/2001/HTML/silverbook/frame1.1.shtml" target="_blank" class="fs14">(1995)</A>. Additional useful literature links are provided in the <B>References</B> link at the top center of this page.
</P>
